data_collection/web/guides.rst

```diff
@@ -515,7 +515,7 @@ The Piwik PRO `Outlinks` report shows the list of external URLs that were clicke
 Configuring which domains are outlinks
 """"""""""""""""""""""""""""""""""""""
 
-When, for example your main page is `piwik.pro` and you want to track views of `help.piwik.pro` without additional outlink click, you have to confgure tracker to recognize this additional domain. You can do it in two ways.
+When, for example, your main page is `piwik.pro` and you want to track views of `help.piwik.pro` without additional outlink click, you have to confgure the tracking client to recognize this additional domain. You can do it in two ways.
```

Comment:
Shouldn't we stick with the proper name we gave to the JS tracker?
```suggestion
When, for example, your main page is `piwik.pro` and you want to track views of `help.piwik.pro` without additional outlink click, you have to confgure the JS Tracking Client to recognize this additional domain. You can do it in two ways.
```

Reply:
Ultimately changed to ' confgure `JS Tracking Client` to recognize'. And changed JS Tracker to proper name in few other places.